book/website/community-handbook/bookdash/bookdash-application.md

```diff
@@ -7,41 +7,46 @@ In this subchapter, we have provided details on the book dash's application and
 
 ## Call for applications
 
-The book dash event is organised for up to 15 participants.
-The reason behind keeping it small is to allow the core team members to offer specific mentorship to the invited participants and make this an interactive, intimate, fun, and productive event.
+The book dash event is organised for a minimum of 15 participants.
+The event is divided into multiple sessions with small groups of participants attending in their time zone.
+This allows the core team members to offer specific mentorship to the invited participants and make this an interactive, intimate, fun, and productive event.
 
-To fairly make this selection, a call for applications is opened at least 3 months in advance so that interested members can share their skills, interest, and expected outcomes through a short application.
+To fairly make this selection, a call for applications is opened at least 2 months in advance so that interested members can share their skills, interest, and expected outcomes through a short application.
```

Comment:
```suggestion
To fairly make this selection, a call for applications is opened at least two months in advance so that interested members can share their skills, interest, and expected outcomes through a short application.
```